[2020-11-26T11:28:50.232] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /api/server/stop
request client ip:  ::ffff:127.0.0.1
request body: 
{"id":150,"status":1,"create_user_id":79,"update_user_id":null,"create_time":"2020-11-12T03:30:33.660Z","update_time":"2020-11-12T03:30:33.660Z","servername":"123123","ip_port":"127.0.0.1:24024","address":"","plaform":["1"],"display":"3","srttime":"2020-11-12T03:23:50.000Z","gamename":"","pid":null,"serverid":"1","test":"0","load":"2","gameid":"1","childrens":null,"port":"24024","ip":"127.0.0.1","channel":["fancyguo"],"serverTrue":false,"securityGroup":null,"clientshow":"0","stopshow":"0","stopmessage":null,"stopuser":null,"showstatusIsShow":false,"because":"阿巴阿巴阿巴"}
response time: 1606361330231
err name: SequelizeDatabaseError
err message: syntax error at or near "stopbecause"
err stack: SequelizeDatabaseError: syntax error at or near "stopbecause"
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee8$ (H:\致命代码\fancyguo-gm\server\src\app\service\/Components.js:190:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at components.<anonymous> (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at components.stopserver (H:\致命代码\fancyguo-gm\server\src\app\service\Components.js:608:28)
    at _callee5$ (H:\致命代码\fancyguo-gm\server\src\app\controllers\/server.js:64:33)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at stopserver (H:\致命代码\fancyguo-gm\server\src\app\controllers\server.js:252:28)
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:48:11)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:43:2
*************** error log end ***************

[2020-11-26T11:29:13.857] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /api/server/stop
request client ip:  ::ffff:127.0.0.1
request body: 
{"id":150,"status":1,"create_user_id":79,"update_user_id":null,"create_time":"2020-11-12T03:30:33.660Z","update_time":"2020-11-12T03:30:33.660Z","servername":"123123","ip_port":"127.0.0.1:24024","address":"","plaform":["1"],"display":"3","srttime":"2020-11-12T03:23:50.000Z","gamename":"","pid":null,"serverid":"1","test":"0","load":"2","gameid":"1","childrens":null,"port":"24024","ip":"127.0.0.1","channel":["fancyguo"],"serverTrue":false,"securityGroup":null,"clientshow":"0","stopshow":"0","stopmessage":null,"stopuser":null,"showstatusIsShow":false,"because":"阿巴阿巴阿巴"}
response time: 1606361353856
err name: SequelizeDatabaseError
err message: argument of AND must be type boolean, not type integer
err stack: SequelizeDatabaseError: argument of AND must be type boolean, not type integer
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee8$ (H:\致命代码\fancyguo-gm\server\src\app\service\/Components.js:190:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at components.<anonymous> (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at components.stopserver (H:\致命代码\fancyguo-gm\server\src\app\service\Components.js:608:28)
    at _callee5$ (H:\致命代码\fancyguo-gm\server\src\app\controllers\/server.js:64:33)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at stopserver (H:\致命代码\fancyguo-gm\server\src\app\controllers\server.js:252:28)
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:48:11)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:43:2
*************** error log end ***************

